app/livechat/client/views/app/tabbar/customerChatHistory.html

```diff
@@ -0,0 +1,40 @@
+<template name='customerChatHistory'>
+	<div class="content">
+		<div class="list-view">
+			<div>
+				<div class="active-pink-4 mb-4 inner-addon right-addon">
+					<i class="glyphicon glyphicon-search icon-search"></i>
+					<input type="text" id="searchInput"  class="" placeholder="Search" aria-label="Search">
+				</div>
+			</div>
+			{{#if hasChatHistory}}
+				{{#if isAllChat}}
+				<div class="vistior-history-block">
+					<ul id='allist'>
+					{{#if isSearching}}
+						{{#if isfound }}
+							{{#each room in searchResults}} 
+								{{> chatRoomSearchItem room}}
```

Comment:
You NEED To use the same template -> `chatRoomHistoryItem` you're search in the `customerChatHistory` template.
Here, the user wants to see the ROOMS that have a message that matches the filter.